src/views/GameListView.jsx

```diff
@@ -150,18 +253,62 @@ const GameListView = () => {
       <Content>
         <Filter>
           <h2 className="incentive">{t("Start a game now")}</h2>
+          <input
+            type="search"
+            id="game-search"
+            name="game-search"
+            aria-label={t("Search for a game")}
+            placeholder={t("Search for a game")}
+            value={filterCriteria.searchTerm}
+            onChange={onChangeSearchTerm}
+          />
           <StyledGameFilters>
             <li>
-              <input
-                type="search"
-                id="game-search"
-                name="game-search"
-                aria-label={t("Search for a game")}
-                placeholder={t("Search for a game")}
-                value={searchTerm}
-                onChange={handleChange}
+              <span className="filter-title">{t("Number of players")}</span>
+              <SliderRange
+                defaultValue={[1, 9]}
+                value={[
+                  filterCriteria.nbOfPlayersMin,
+                  filterCriteria.nbOfPlayersMax,
+                ]}
+                step={1}
+                onChange={onChangeNbOfPlayersSlider}
               />
             </li>
+            <li>
+              <span className="filter-title">{t("Duration (mins)")}</span>
+              <SliderRange
+                defaultValue={[15, 90]}
+                value={[filterCriteria.durationMin, filterCriteria.durationMax]}
+                step={15}
+                onChange={onChangeDurationSlider}
+              />
+            </li>
+            <li className="small-filter">
+              <span className="filter-title">{t("Language")}</span>
+              <div className="language-filters">
+                <ul>
+                  <li>
+                    <input
+                      type="checkbox"
+                      id="french-filter"
+                      name="french-filter"
+                      defaultChecked={filterCriteria.language.fr}
+                    ></input>
```

Comment:
With the state modification it becomes:

```suggestion
                    <input
                      type="checkbox"
                      id="french-filter"
                      name="french-filter"
                      defaultChecked={filterCriteria.languages.includes("fr")}
                    />
```

Reply:
But you have to use a controlled input then.

Reply:
But seems broken in the preview...